RadarSDK/RadarLocationManager.m

```diff
@@ -461,12 +586,18 @@ - (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
     NSUInteger numGeofences = MIN(geofences.count, 9);
     NSMutableArray *requests = [NSMutableArray array]; 
 
+    // bool for whether or not we're within the ramp up radius of a geofence
+    BOOL withinRampUpRadius = NO;
+    RadarTrackingOptions *trackingOptions = [Radar getTrackingOptions];
+
+
     for (int i = 0; i < numGeofences; i++) {
         RadarGeofence *geofence = [geofences objectAtIndex:i];
         NSString *geofenceId = geofence._id;
         NSString *identifier = [NSString stringWithFormat:@"%@%@", kSyncGeofenceIdentifierPrefix, geofenceId];
         RadarCoordinate *center;
         double radius = 100;
+        bool rampUpGeofenceAndDifferentFromOriginalGeofence = NO;
```

Comment:
A little confused by the naming of this variable? Can you share a little more context? Thanks!

Reply:
Yea, `rampUpGeofenceAndDifferentFromOriginalGeofence` is meant to track whether or not we should be populating local notifications onto the synced geofence. If the geofence has the geometry of the original geofence synced, the inverse of which `rampUpGeofenceAndDifferentFromOriginalGeofence` is meant to signify, then we can sync the notifications, else we need to skip.

Reply:
I'm gonna rename to something like `skipSyncingLocalNotifications`